src/runtime-rs/crates/hypervisor/src/qemu/cmdline_generator.rs

```diff
@@ -1299,7 +1352,7 @@ impl<'a> QemuCmdLine<'a> {
             smp: Smp::new(config),
             machine: Machine::new(config),
             cpu: Cpu::new(config),
-            qmp_sockets: QmpSocket::new()?,
+            qmp_sockets: QmpSocket::new(id, false)?,
```

Comment:
I believe that the Qmp sockets should go to the anonymous `devices` container since QemuCmdLine doesn't need to interact with them anymore once they are created (@gkurz ?) so there's no need to hold them by name for future reference.